subsys/net/lib/http/Kconfig

```diff
@@ -123,6 +123,16 @@ config HTTP_SERVER_RESOURCE_WILDCARD
 	  This means that instead of specifying multiple resources with exact
 	  string matches, one resource handler could handle multiple URLs.
 
+config HTTP_SERVER_RESTART_DELAY
+	int "Delay before re-initialization when restarting server"
+	default 1000
+	range 0 60000
```

Comment:
This still feels error-prone as I read the description, would 0 ever be desired?

Reply:
Possibly not, although with this range I just wanted to ensure no one sets a negative delay. I want to be able to set a very small delay here, to verify the feature with ztest (I'm working on extending the server test coverage).

Reply:
On second thought, I've changed the lower range to 1, just in case. Setting 0 could indeed have nasty consequences if initialization fails. Thanks for pointing that out.